bobtemplates/plone_addon/src/+package.namespace+/+package.name+/tests/test_+package.dexterity_type_name_lower+.py.bob

```diff
@@ -1,15 +1,14 @@
 # -*- coding: utf-8 -*-
-from plone.app.testing import TEST_USER_ID
-from zope.component import queryUtility
-from zope.component import createObject
+from plone import api
 from plone.app.testing import setRoles
+from plone.app.testing import TEST_USER_ID
 from plone.dexterity.interfaces import IDexterityFTI
-from plone import api
-
-from {{{ package.dottedname }}}.testing import {{{package.uppercasename}}}_INTEGRATION_TESTING  # noqa
 from {{{ package.dottedname }}}.interfaces import I{{{ package.dexterity_type_name }}}
+from {{{ package.dottedname }}}.testing import {{{package.uppercasename}}}_INTEGRATION_TESTING  # noqa
+from zope.component import createObject
+from zope.component import queryUtility
 
-import unittest2 as unittest
+import unittest
```

Comment:
bobtemplates.plone generated packages do not have a troove classifier for Python 2.6, they state to be for 2.7:
https://github.com/plone/bobtemplates.plone/blob/master/bobtemplates/plone_addon/setup.py.bob#L26